lib/open_api_spex/cast/discriminator.ex

```diff
@@ -70,20 +70,21 @@ defmodule OpenApiSpex.Cast.Discriminator do
 
   defp cast_composition(%_{schema: %{anyOf: schemas, discriminator: nil}} = ctx)
        when is_list(schemas),
-       do: {schemas, Cast.cast(ctx)}
+       do: {schemas |> _locate_schemas(ctx.schemas), Cast.cast(ctx)}
```

Comment:
Can this be updated so that it's not a pipeline expression? For guidance, here's a popular Elixir style guide and its guideline around pipe operators: https://github.com/christopheradams/elixir_style_guide#avoid-single-pipelines

Can you apply the same style change to the rest of the changes?